### PR TITLE
fix: update firefox version to prevent crashes

### PR DESCRIPTION
**Pull request by ca-scribner**
Bump firefox to current version.  I've tested briefly and the previous version (76-something) crashed **very** frequently (on ~50% page loads) while the new version lasted for 10 minutes of testing.

### CODE REVIEW

**Review by Jose-Matsuda — CHANGES_REQUESTED** [review]
Just concerning i18n on line [742](https://github.com/StatCan/kubeflow-containers-desktop/blob/90a6ae3e07b7a20c9b494d2187df514d277be844/base/Dockerfile#L742) it installs the French language pack. It may still work / i18nize but not 100% sure, can test just by adding `ENV KF_LANG=fr` somewhere and if that xpi package works it works. 

Else
Now in that same url directory I could not find an `xpi` file for firefox 86 but found one here
https://ftp.mozilla.org/pub/firefox/nightly/2020/12/2020-12-31-21-16-52-mozilla-central-l10n/linux-x86_64/xpi/

`https://ftp.mozilla.org/pub/firefox/nightly/2020/12/2020-12-31-21-16-52-mozilla-central-l10n/linux-x86_64/xpi/firefox-86.0a1.fr.langpack.xpi

I'm pretty sure that any language pack with 86 should work (which again can test easily with KF_LANG=fr and opening up the web browser

TLDR; Replace the wget target url with that one above?

**Comment by ca-scribner** [comment]
We can also back the version off a point or two if that’s easier for i18n.
Pick whichever you prefer and give it a go! :D

**Comment by Jose-Matsuda** [comment]
Scratch that above URL out of your memory, it will not work (tried on personal installation of firefox)
https://addons.mozilla.org/en-US/firefox/addon/fran%C3%A7ais-language-pack/versions/

I was concerned with the fact that the URL would disappear but I found a URL from an older version (here is for [68](https://addons.mozilla.org/firefox/downloads/file/3379574/francais_language_pack-68.0buildid20190813150448-fx.xpi))of Firefox linked in an issue [here](https://github.com/MrAlex94/Waterfox/issues/1055).


So what I will do is replace that line with the following instead `https://addons.mozilla.org/firefox/downloads/file/3731010/francais_language_pack-86.0buildid20210222142601-fx.xpi`

[Streamable link](https://streamable.com/mjdzqo)

**Comment by Jose-Matsuda** [comment]
This is now a 'LGTM' from me
